admin/views/gds-layout.ejs

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template">
+  <%- partial('partials/_gds_head') %>
+
+  <body class="govuk-template__body">
+    <script>
+      document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+    </script>
+
+    <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
+
+    <%- partial('partials/_body_start') %>
+
+    <%- partial('partials/_gds_cookie_message') %>
+
+    <%- partial('partials/_gds_body_header') %>
+
+    <%- partial('partials/_gds_content', {body: body}) %>
+
+    <%- partial('partials/_gds_body_footer') %>
+
+    <div id="global-app-error" class="app-error hidden"></div>
+
+    <script src="/javascripts/app.js"></script>
+
+    <script>
+      window.GOVUKFrontend.initAll()
```

Comment:
This makes us require all the GDS frontend JS synchronously.  But the same is not necessarily true for our other JS.  We should split it out from our code, just because of this.  Our code could then be loaded using async.